kyaml/setters2/add.go

```diff
@@ -176,6 +200,24 @@ func (sd SetterDefinition) Filter(object *yaml.RNode) (*yaml.RNode, error) {
 	return object, nil
 }
 
+// ConvertJSONToYamlNode parses input json string and returns equivalent yaml node
+func ConvertJSONToYamlNode(jsonStr string) (*yaml.RNode, error) {
```

Comment:
Maybe this could be moved into somewhere where it could be reused? And are we sure we don't have this functionality implemented already?

Reply:
I don't see it's existing already. Moved to yaml library.